tests/page-objects/dataTable.ts

```diff
@@ -0,0 +1,77 @@
+import { Page, Locator, expect } from "@playwright/test"
+import { Dialog } from "puppeteer";
```

Comment:
Why are you pulling in a completely different library? I don't think you need to do this.